deep_agent/aegra/lifecycle.py

```
@@ -0,0 +1,685 @@
+                    )
+
+                    try:
+                        await resume_fn(run_id, thread_id)
+                        await cur.execute(
+                            "UPDATE runs SET status = 'success', "
+                            "updated_at = NOW() WHERE run_id = %s",
+                            (run_id,),
+                        )
+                        results[run_id] = "resumed"
+                        logger.info("Resumed interrupted run %s", run_id)
+                    except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)